app/__test__/unit.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Generator } from "../utils";
+
+test('returns instance of Generator', () => {
+  expect(new Generator).toBeInstanceOf(Generator);
+});
+
+test('should have getId method ', () => {
+  expect(new Generator()).toHaveProperty('getId');
+});
+
+test('getId should be a function', () => {
+  expect(typeof new Generator().getId).toBe('function');
+});
```

Comment:
I don't think these tests are helpful unless there is a requirement to implement a particular `interface`.